examples/ts/sample-local-proxy-server-with-cors.ts

```diff
@@ -0,0 +1,11 @@
+// @ts-nocheck
+// JavaScript CORS Proxy
+// Save this in a file like cors.js and run with:
+//    node cors [port]
+// It will listen for your requests on the port you pass in command line (or port 8080 by default)
+
+import cors from 'cors-anywhere';
+
+const port = (process.argv.length > 2) ? parseInt (process.argv[2]) : 8080;
```

Comment:
Hum I don't know if this is useful or not here, since is not "ccxt-related"

Reply:
the thing is that, whether or not it's ccxt-related, we have that in "install.md" at this moment :) https://github.com/ccxt/ccxt/blob/master/wiki/Install.md#nodejs-cors-proxy

so, this move is much more better than having it in readme or install (which is really unrelated to have it in readme). and, as a considerable exclusion, we can consider it ccxt-related, because users using ccxt from browser will need "extra" thing to make ccxt work, so it would benefit in ccxt related scope.

Reply:
to add another argument, we already have it in examples (in a "non-unifiish" name) https://github.com/ccxt/ccxt/blob/master/examples/js/bitmex-browser-cors-proxy.js